### PR TITLE
E2E-test to verify apply with CRD and CR

### PR DESCRIPTION
**Pull request by mortent**
/hold

### CODE REVIEW

File: examples/alphaTestExamples/crds.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+Alternatively, use
+
+> ```
+> DEMO_HOME=~/hello
+> ```
+
+## Establish the base
+
+Create the a CRD and a CR.
```
Comment (phanimarupaka) [inline examples/alphaTestExamples/crds.md:24]: Nit: the a

Comment (mortent) [reply examples/alphaTestExamples/crds.md]: Fixed

**Pull request merged**